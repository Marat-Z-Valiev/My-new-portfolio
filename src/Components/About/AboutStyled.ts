import styled from "styled-components";

const AboutStyled = styled.div`
	display: flex;
	width: auto;
	flex-direction: column;
	align-items: center;
	margin-top: 130px;

	img {
		width: 200px;
		height: 200px;
		border-radius: 50%;
	}
`;

export default AboutStyled;
