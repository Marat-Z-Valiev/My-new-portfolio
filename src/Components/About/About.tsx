import React from "react";
import AboutStyled from "./AboutStyled";

const About: React.FC = () => {
	return (
		<>
			<AboutStyled>
				<img src="https://i.ibb.co/FnHD0NB/portfolio.jpg" alt="" />
				<h2>Marat Valiev</h2>
				<h2>Web Developer</h2>
			</AboutStyled>
		</>
	);
};

export default About;
