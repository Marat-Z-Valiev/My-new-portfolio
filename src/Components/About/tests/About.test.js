import React from "react";
import {shallow, configure} from "enzyme";
import toJSON from "enzyme-to-json";
import Adapter from "enzyme-adapter-react-16";
import About from "../About";

configure({adapter: new Adapter()});

describe("About", () => {
	it("render correctly", () => {
		const wrapper = shallow(<About />);
		expect(toJSON(wrapper)).toMatchSnapshot();
	});
});
