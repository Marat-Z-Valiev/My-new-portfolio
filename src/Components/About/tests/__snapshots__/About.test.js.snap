// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About render correctly 1`] = `
<Fragment>
  <AboutStyled>
    <img
      alt=""
      src="https://i.ibb.co/FnHD0NB/portfolio.jpg"
    />
    <h2>
      Marat Valiev
    </h2>
    <h2>
      Web Developer
    </h2>
  </AboutStyled>
</Fragment>
`;
